src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -97,15 +97,14 @@ public override ExportResult Export(in Batch<Metric> batch)
                         var bucketsBuilder = new StringBuilder();
                         var sum = metricPoint.GetHistogramSum();
                         var count = metricPoint.GetHistogramCount();
+                        bucketsBuilder.Append($"Sum: {sum} Count: {count} ");
                         if (metricPoint.HasMinMax())
                         {
-                            bucketsBuilder.Append($"Sum: {sum} Count: {count} Min: {metricPoint.GetHistogramMin()} Max: {metricPoint.GetHistogramMax()} \n");
-                        }
-                        else
-                        {
-                            bucketsBuilder.Append($"Sum: {sum} Count: {count} \n");
+                            bucketsBuilder.Append($"Min: {metricPoint.GetHistogramMin()} Max: {metricPoint.GetHistogramMax()} ");
                         }
 
+                        bucketsBuilder.Append("\n\r");
```

Comment:
Myave AppendLine? Library should handle correct endline for us. 
```suggestion
                        bucketsBuilder.AppendLine();
```